### ISSUE TITLE
train on flux, how to get rgb image on stage 2?

### ISSUE THREAD

**Issue by Shallo2512** [ToTheBeginning/PuLID#191]
Nice work!
When train on Flux, I would ask which lightweight network should be used to obtain RGB images when training a 2-stage network? 
Thanks.

**Comment by csjunxu** [comment]
hi，may i ask you how to train the model on our own?

**Comment by LeonNerd** [comment]
I am also trying to train the model, but I have some confusion regarding the loss design in the second stage. According to the paper, the lighting branch generates a prediction image from pure noise in 4 steps and calculates the ID loss with the original image. This process is not influenced by the diffusion loss branch's tempstep, but rather it is affected only by the most recently updated pulid encoder and pulid_ca models. By adding these two losses together for backpropagation, does the ID loss indirectly affect the gradient updates? Currently, I am quite confused as the model is converging very slowly.
